src/main/java/com/notably/logic/parser/NewCommandParser.java

```diff
@@ -37,13 +37,20 @@ public NewCommandParser(Model notablyModel) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE, PREFIX_BODY, PREFIX_JUMP);
 
-        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE, PREFIX_BODY)
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format("Invalid Command"));
         }
 
+
         String title = argMultimap.getValue(PREFIX_TITLE).get();
-        String body = argMultimap.getValue(PREFIX_BODY).get();
+        String body;
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_BODY)
+                || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
You don't really need the `|| !argMultimap.getPreamble().isEmpty()` portion, I believe?

Reply:
oh yes, since it was checked before. Thanks for pointing it out